Dockerfile

```diff
@@ -9,4 +9,5 @@ COPY ./requirements.txt requirements.txt
 RUN pip install --no-cache-dir -r requirements.txt
```

Comment:
create a separate requirements-production.txt file. we don't want or need to install pytest in the deployable container.

Reply:
done